xrpl4j-core/src/main/resources/definitions.json

```diff
@@ -260,6 +262,16 @@
         "type": "UInt8"
       }
     ],
+    [
+      "AssetScale",
+      {
+        "nth": 20,
```

Comment:
I think this value is `5` in [the PR](https://github.com/XRPLF/rippled/pull/5143/files#diff-5b935a0aed0713949821a5aa5e4daaf78e3648db69d065530338035c39ffa4edR38).

Reply:
xrpl.js must have this wrong...